scripts/build-on-travis.sh

```diff
@@ -26,7 +26,7 @@ if test "x$BUILD_WITH_CMAKE" = "xyes"; then
   cd build
   time cmake -Wdev -GNinja -C../cmake/caches/Travis.cmake ..
   if test "x$RUN_BUILD" != "xno"; then
-    time ninja -j1  
+    time ninja -j$JOBS  
```

Comment:
@dnakamura didn't you have an issue with ninja files and parallel compiles in the past?